graphics/ctpv/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github   1.0
+PortGroup           makefile 1.0
+
+github.setup        NikitaIvanovV ctpv 6f364cf383e87d2ab51bee307bbd149c2fb252db
```

Comment:
please use the released version 1.0 not the latest commit. One should typically package the latest released version unless there is a pressing reason for not to do so; if you want a `-devel` subport could possibly be added to provided the unreleased version based on the latest commit.

Reply:
[ctpv 1.1](https://github.com/NikitaIvanovV/ctpv/releases/tag/v1.1) just released today. This can now be resolved.